2022-02-08 21:46:46,213 INFO    MainThread:423389 [wandb_setup.py:_flush():71] setting env: {}
2022-02-08 21:46:46,213 INFO    MainThread:423389 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-08 21:46:46,213 INFO    MainThread:423389 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_214646-3cenyxua/logs/debug.log
2022-02-08 21:46:46,213 INFO    MainThread:423389 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_214646-3cenyxua/logs/debug-internal.log
2022-02-08 21:46:46,213 INFO    MainThread:423389 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f42cc109700>
2022-02-08 21:46:46,214 INFO    MainThread:423389 [wandb_init.py:init():404] calling init triggers
2022-02-08 21:46:46,214 INFO    MainThread:423389 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-08 21:46:46,214 INFO    MainThread:423389 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 3f3dasn8
2022-02-08 21:46:46,216 INFO    MainThread:423389 [wandb_run.py:finish():1457] finishing run nelsonsw5/3DDet_Viz/3f3dasn8
2022-02-08 21:46:46,216 INFO    MainThread:423389 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-08 21:46:46,218 INFO    MainThread:423389 [wandb_run.py:_restore():1752] restore
2022-02-08 21:46:46,631 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 21:46:46,761 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 21:46:47,048 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 21:46:48,028 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11648
}

2022-02-08 21:46:48,132 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11648
}

2022-02-08 21:46:48,234 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11648
}

2022-02-08 21:46:48,337 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11648
}

2022-02-08 21:46:48,439 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4778
  total_bytes: 11648
}

2022-02-08 21:46:48,543 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11648
  total_bytes: 11648
}

2022-02-08 21:46:48,648 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11648
  total_bytes: 11648
}

2022-02-08 21:46:48,750 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11648
  total_bytes: 11648
}

2022-02-08 21:46:48,852 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11648
  total_bytes: 11648
}

2022-02-08 21:46:48,992 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11648
  total_bytes: 11648
}

2022-02-08 21:46:49,589 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11648
  total_bytes: 11648
}
local_info {
}

2022-02-08 21:46:50,713 INFO    MainThread:423389 [wandb_run.py:_append_files():2180] logging synced files
2022-02-08 21:46:50,720 INFO    MainThread:423389 [wandb_init.py:init():460] starting backend
2022-02-08 21:46:50,720 INFO    MainThread:423389 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-08 21:46:50,726 INFO    MainThread:423389 [backend.py:ensure_launched():216] starting backend process...
2022-02-08 21:46:50,740 INFO    MainThread:423389 [backend.py:ensure_launched():221] started backend process with pid: 424252
2022-02-08 21:46:50,741 INFO    MainThread:423389 [wandb_init.py:init():469] backend started and connected
2022-02-08 21:46:50,761 INFO    MainThread:423389 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-08 21:46:50,761 INFO    MainThread:423389 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-08 21:46:50,761 INFO    MainThread:423389 [wandb_init.py:init():533] updated telemetry
2022-02-08 21:46:50,788 INFO    MainThread:423389 [wandb_init.py:init():563] communicating current version
2022-02-08 21:46:51,402 INFO    MainThread:423389 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-08 21:46:51,404 INFO    MainThread:423389 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-08 21:46:51,663 INFO    MainThread:423389 [wandb_init.py:init():606] starting run threads in backend
2022-02-08 21:46:53,756 INFO    MainThread:423389 [wandb_run.py:_console_start():1810] atexit reg
2022-02-08 21:46:53,757 INFO    MainThread:423389 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-08 21:46:53,757 INFO    MainThread:423389 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-08 21:46:53,760 INFO    MainThread:423389 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-08 21:46:53,760 INFO    MainThread:423389 [wandb_init.py:init():633] run started, returning control to user process
2022-02-08 21:46:53,760 INFO    MainThread:423389 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-08 21:46:53,820 INFO    MainThread:423389 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 21:46:53,820 INFO    MainThread:423389 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 21:47:07,305 INFO    MainThread:423389 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 21:47:07,322 INFO    MainThread:423389 [wandb_setup.py:_flush():71] setting env: {}
2022-02-08 21:47:07,322 INFO    MainThread:423389 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-08 21:47:07,323 INFO    MainThread:423389 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_214707-37ywkqm5/logs/debug.log
2022-02-08 21:47:07,323 INFO    MainThread:423389 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_214707-37ywkqm5/logs/debug-internal.log
2022-02-08 21:47:07,323 INFO    MainThread:423389 [wandb_init.py:init():404] calling init triggers
2022-02-08 21:47:07,323 INFO    MainThread:423389 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-08 21:47:07,323 INFO    MainThread:423389 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 3cenyxua
2022-02-08 21:47:07,325 INFO    MainThread:423389 [wandb_run.py:finish():1457] finishing run nelsonsw5/3DDet_Viz/3cenyxua
2022-02-08 21:47:07,325 INFO    MainThread:423389 [jupyter.py:save_history():429] not saving jupyter history
2022-02-08 21:47:07,325 INFO    MainThread:423389 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 21:47:07,325 INFO    MainThread:423389 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-08 21:47:07,325 INFO    MainThread:423389 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-08 21:47:07,327 INFO    MainThread:423389 [wandb_run.py:_restore():1752] restore
2022-02-08 21:47:07,788 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 21:47:07,907 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 21:47:08,218 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 21:47:08,689 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11816
}

2022-02-08 21:47:08,793 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11842
}

2022-02-08 21:47:08,900 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11842
}

2022-02-08 21:47:09,002 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11842
}

2022-02-08 21:47:09,104 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8454
  total_bytes: 11842
}

2022-02-08 21:47:09,208 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11842
  total_bytes: 11842
}

2022-02-08 21:47:09,311 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11842
  total_bytes: 11842
}

2022-02-08 21:47:09,414 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11842
  total_bytes: 11842
}

2022-02-08 21:47:09,516 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11842
  total_bytes: 11842
}

2022-02-08 21:47:09,940 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11842
  total_bytes: 11842
}

2022-02-08 21:47:10,492 INFO    MainThread:423389 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11842
  total_bytes: 11842
}
local_info {
}

2022-02-08 21:47:11,617 INFO    MainThread:423389 [wandb_run.py:_append_files():2180] logging synced files
